extensions/sns/src/utils.rs

```diff
@@ -0,0 +1,15 @@
+use anyhow::{anyhow, Result};
+use ic_agent::agent::Agent;
+
+fn get_agent(ic_url: &str) -> Result<Agent> {
+    Agent::builder()
+        .with_url(ic_url)
+        .with_verify_query_signatures(false)
```

Comment:
I'm not really sure what this does, but it looks scary, like leaving your house without locking the door. Sure, if there are no evil doers around, everything works just fine. But you still wouldn't do that.

Reply:
Okay, done